LiveLite/recommendation_tool/diet_recommendation/nutrients.py

```diff
@@ -0,0 +1,91 @@
+import pandas as pd
+import numpy as np
+
+food = pd.read_csv("../NutritionData/nutrition_data1.csv")
```

Comment:
Perhaps reconsider using global variables. The file read could be moved into a function (see file comment).